src/config/config.go

```diff
@@ -77,6 +77,9 @@ var (
 
 	// Dev specifies if we are running in dev mode
 	Dev = false
+
+	// Map of refs specified for dev deploy
+	DevDeployRefs map[string]string
```

Comment:
Should this be in `config` or maybe in the DeployOpts struct?

Reply:
Don't love having to define it here, but needed to be able to access it [here](https://github.com/defenseunicorns/uds-cli/pull/638/files#diff-cc6b07ec48af179448d51a500be72923462835fd2e22b0f5d6b49667c5be4ac9R49), at which point I don't have access to Bundle.BundleConfig.DevDeployOpts

Reply:
What do you think of adding the entire[ b.cfg](https://github.com/defenseunicorns/uds-cli/blob/8f46e31aa4803b4a44a1eda2eb466a50f5073988/src/pkg/bundle/deploy.go#L128) to the `TarballBundle` and `RemoteBundle` structs? As in, modify that `sources.New` function to take `b.cfg` instead of just the source, then we can handle `DeployOpts` and `DevDeployOpts` in the sources

Reply:
the only potential issue I see with that is that sources.New gets called on deploy and remove. Would need to know where to grab the source from,  from within the sources.New function (from DeployOpts.Source or RemoveOpts.Source)